nodeup/pkg/model/kube_apiserver.go

```diff
@@ -347,6 +347,13 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 		}
 	}
 
+	//remove elements from the spec that are not enabled yet
+	if b.Cluster.Spec.KubeAPIServer.AuditDynamicConfiguration != nil {
```

Comment:
`if b.Cluster.Spec.KubeAPIServer.AuditDynamicConfiguration != nil && !b.IsKubernetesGTE("1.13") {`

Reply:
ok, I'll push an update after this and rebasing with the upstream master again

Reply:
I updated bazel for the test runtime environment and this style update.